docs/README.md

```diff
@@ -1,115 +1,346 @@
-## Getting started
+# Getting started
 
-### Prequisites
+* [Installation](#installation)
+  * [Prerequisites](#prerequisites)
+    * [MacOS](#macos)
+    * [Windows](#windows)
+    * [Linux](#linux)
+  * [Getting the CLI](#getting-the-cli)
+  * [Using the unstable version](#using-the-unstable-version)
+* [Usage](#usage)
+  * [Target an AMP cluster](#target-an-amp-cluster)
+    * [Creating a local AMP cluster](#creating-a-local-amp-cluster)
+  * [Signing up and logging in](#signing-up-and-logging-in)
+  * [UI and Dashboard](#ui-and-dashboard)
+    * [UI](#ui)
+    * [Grafana](#grafana)
+  * [Examples](#examples)
+    * [Deploying a stack](#deploying-a-stack)
+    * [Viewing and filtering logs](#viewing-and-filtering-logs)
+    * [Displaying metrics and statistics](#displaying-metrics-and-statistics)
+    * [Organization and team management](#organization-and-team-management)
 
-We are currently targeting the next stable release of Docker, anticipated to be Docker 17.06.
-For now, we recommend installing the latest Docker CE edge channel release (17.05) on your system:
+## Installation
 
- * [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
- * [Docker for Windows](https://docs.docker.com/docker-for-windows/install/)
- * [Docker for Linux](https://docs.docker.com/engine/installation/) (see specific distribution under the sidebar).
- 
-#### Linux Node:
+### Prerequisites
 
-If you're running on Linux, there are a few steps you must perform for your system in addition to installing Docker.
+The only prerequisite required to run `amp` is Docker 17.03 or greater.
 
-Check if you have IPVS kernel modules loaded:
+We recommend installing the Docker CE stable channel release 17.06 on your system.
+Make sure you check the "What to know before you install" section to ensure your system meets the requirements.
```

Comment:
The `Docker for Linux` installation doesn't have a `What to know before you install` section. There is on both the macOS and Windows links. There is also a section in the sidebar called `Post-installation steps for Linux` which we could point the users towards?